roles/ci_gen_kustomize_values/templates/nfv/edpm-values/values.yaml.j2

```diff
@@ -0,0 +1,61 @@
+{% set instances_names = []                                                            %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _vm_type = (_original_nodes.keys() | first).split('-')[1]                       %}
+{% for _inst in cifmw_networking_env_definition.instances.keys()                       %}
+{%   if _inst.startswith(_vm_type)                                                     %}
+{%     set _ = instances_names.append(_inst)                                           %}
+{%   endif                                                                             %}
+{% endfor                                                                              %}
+data:
+  preProvisioned: false
+  baremetalSetTemplate:
+{% for instance in instances_names                                                     %}
+    ctlplaneInterface: "{{ cifmw_networking_env_definition.instances[instance].networks.ctlplane.interface_name }}"
+    cloudUserName: cloud-admin
+    provisioningInterface: ## TODO!
+    bmhLabelSelector:
+      app: openstack
```

Comment:
also here, i left this label as default, if we want to customize it, we should pass the value from elsewhere

Reply:
I think we will be fine using this one as a starting point.